build.sbt

```diff
@@ -8,21 +8,22 @@ lazy val `play-mailer` = (project in file("."))
     scalaVersion := scala212,
     crossScalaVersions := Seq(scala211, scala212)
   )
-    
+
 val PlayVersion = playVersion("2.6.0-RC2")
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play" % PlayVersion % Provided,
+  "com.google.inject" % "guice" % "4.0", // 4.0 to maybe make it work with 2.5 and 2.6
```

Comment:
Currently for Play 2.5.15 I also need to have bindings for `Config`, however after this PR I want to split the project into play-mailer and play-mailer-guice, and I can maybe add another play-mailer-compat which creates the Config bindings, but is not provided by default (because maybe somebody already have these bindings)